google/cloud/forseti/services/client.py

```diff
@@ -129,6 +131,66 @@ def run(self):
                              metadata=self.metadata())
 
 
+class ServerConfigClient(ForsetiClient):
+    """Server configuration service allows the client to update the
```

Comment:
This should be one line.  How about `Allows the client to update the server configuration.`